docs/did-button-branding/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: "doc"
+title: "Branding DID.app buttons."
+abstract: "DID.app provides several styles of icon that you can use to let people sign in."
+---
+
+<style>
+  .markdown-body img {
+    background: none;
+  }
+</style>
+
+You can use either our "D.I.D Logo" or "D Icon" to indicate to your users that they may sign in with DID.app.
+Both come in four styles; **default**, **light**, **dark-mono** and **light-mono**.
+
+### The D.I.D Logo
+
+<div style="display:flex;justify-content:center;">
+  <img src="/logo.svg" style="height:50px;margin:0.5em;padding:0.5em;">
+  <img src="/logo.svg#light" style="height:50px;background:#2a454e;margin:0.5em;padding:0.5em;">
+  <img src="/logo.svg#light-mono" style="height:50px;background:#00dfc0;margin:0.5em;padding:0.5em;">
+  <img src="/logo.svg#dark-mono" style="height:50px;background:#00dfc0;margin:0.5em;padding:0.5em;">
+</div>
+
+```html
+<img src="/logo.svg" style="padding:0.5em;">
+<img src="/logo.svg#light" style="padding:0.5em;background:#2a454e;">
+<img src="/logo.svg#light-mono" style="padding:0.5em;background:#00dfc0;">
+<img src="/logo.svg#dark-mono" style="padding:0.5em;background:#00dfc0;">
+```
+
+### The D Icon
+
+<div style="display:flex;justify-content:center;">
+  <img src="/icon.svg" style="height:50px;margin:0.5em;padding:0.5em;">
+  <img src="/icon.svg#light" style="height:50px;background:#2a454e;margin:0.5em;padding:0.5em;">
+  <img src="/icon.svg#light-mono" style="height:50px;background:#00dfc0;margin:0.5em;padding:0.5em;">
+  <img src="/icon.svg#dark-mono" style="height:50px;background:#00dfc0;margin:0.5em;padding:0.5em;">
+</div>
+
+```html
+<img src="/icon.svg" style="padding:0.5em;">
+<img src="/icon.svg#light" style="padding:0.5em;background:#2a454e;">
+<img src="/icon.svg#light-mono" style="padding:0.5em;background:#00dfc0;">
+<img src="/icon.svg#dark-mono" style="padding:0.5em;background:#00dfc0;">
+```
+
+### Creating buttons
```

Comment:
```suggestion
### When to use DID.app buttons
```

Reply:
Going to make this "How to use DID.app buttons"